src/process/CMakeLists.txt

```
@@ -4,3 +4,4 @@ add_subdirectory(dust)
 add_subdirectory(seasalt)
 add_subdirectory(plumerise)
 add_subdirectory(drydep)
+add_subdirectory(SUVolcanicEmissions)
```

```suggestion
add_subdirectory(volcanic)
```
More consistent with the others? If you agree, should change the module name too.

---

Done!